site/documentation.markdown

```diff
@@ -27,6 +27,7 @@ If you are new to Haskell and are not sure where to start from, we recommend [CI
 ## Intermediate Haskell Books
 
 *   [Developing Web Applications with Haskell and Yesod](http://www.yesodweb.com/book)
+*   \[$$\] [Effective Haskell](https://www.pragprog.com/titles/rshaskell/effective-haskell/)
```

Comment:
Looks great to me. We could make the beta status of the book pretty simply—should be fine as long as we don't forget to update it when it's out of beta!

I reviewed the book in detail earlier and would definitely agree that it could be moved to the beginner section. It covers some intermediate material, but it does not assume any Haskell knowledge and starts by teaching basic Haskell ideas suitable for people learning the language for the first time.

```suggestion
*   \[$$\] [Effective Haskell (beta)](https://www.pragprog.com/titles/rshaskell/effective-haskell/)
```